eZ/Bundle/EzPublishRestBundle/Resources/config/routing.yml

```diff
@@ -95,6 +95,15 @@ ezpublish_rest_copyContent:
     requirements:
         contentId: \d+
 
+ezpublish_rest_deleteContentTranslation:
+    path: /content/objects/{contentId}/{languageCode}
+    defaults:
+        _controller: ezpublish_rest.controller.content:deleteContentTranslation
+    methods: [DELETE]
+    requirements:
+        contentId: \d+
+        languageCode: '[a-z]{3}-[A-Z]{2}'
```

Comment:
No, AFAICS this was the only route for such path and only for `DELETE` method, so wasn't implemented because we didn't have `removeTranslation`. Asked because I'm not sure if I'm allowed to change existing spec in 6.x even for not implemented stuff ;)